pychromecast/socket_client.py

```diff
@@ -281,7 +301,7 @@ def send_message(self, destination_id, namespace, data,
         msg.destination_id = destination_id
         msg.payload_type = cast_channel_pb2.CastMessage.STRING
         msg.namespace = namespace
-        msg.payload_utf8 = json.dumps(data, ensure_ascii=False).encode("utf8")
+        msg.payload_utf8 = json.dumps(data)
```

Comment:
Are you sure this will break?

Won't the protobuf library support strings as well as bytes in the payload_utf8 field?
ie. it will convert (unicode) strings itself?

After testing the json output a bit I think it might be an idea to change the dumps to:

    json.dumps(data, ensure_ascii=False)

That way unicode characters will be encoded to utf-8 and not encoded as the ASCII equivalent, this will be more space efficient.